rfcs/044-sierra-series/README.md

```diff
@@ -0,0 +1,503 @@
+    490 1  Morphogenesis of the vertebrate brain ;|v2
+    830  0 Morphogenesis of the vertebrate brain ;|v2
+    ```
+
+    We'll deduplicate because the information in 490 and 830 is the same:
+
+    ```json
+    "partOf": [
+      {
+        "title": "Morphogenesis of the vertebrate brain; 2",
+        "workType": {
+          "id": "???",
+          "label": "Volumes",
+          "type": "Format"
+        },
+        "partOf": [
+          {
+            "title": "Morphogenesis of the vertebrate brain",
+            "type": "Series"
+          }
+        ],
+        "type": "Series"
+      }
+    ]
+    ```
+
+    We have a nested `partOf` because there's volume information in subfield $v.
+
+
+-   b31098058
+
+    ```
+    490 0  Early European Books : Printed sources to 1700 ;
+           |vCollection 4
+    773 0  |tEarly European Books.
+    ```
+
+    Here we have 490 and 773 with slightly different information.
+
+    ```json
+    "partOf": [
+      { // 490
+        "title": "Early European Books : Printed sources to 170; Collection 4",
+        "workType": {
+          "id": "???",
+          "label": "Volumes",
+          "type": "Format"
+        },
+        "partOf": [
+          {
+            "title": "Early European Books : Printed sources to 170",
+            "type": "Series"
+          }
+        ],
+        "type": "Series"
+      },
+      { // 773
+        "title": "Early European Books",
+        "type": "Work"
+      }
+    ]
+    ```
+
+    Although we could write logic to deduplicate these, it's better to do the deduplication (if we want to) in Sierra itself.
+
+-   b31787
+
+    ```
+    490 1  Perspectives in Continental philosophy series,|x1089-3938
+           ;|vno. 39
+    773 0  |tACLS Humanities E-Book.|nURL: http://
+           www.humanitiesebook.org/
+    830  0 Perspectives in continental philosophy ;|vno. 39.
+    830  0 Fordham perspectives in continental philosophy.
+    830  0 ACLS Humanities E-Book.
+    ```
+
+    ```json
+    "partOf": [
+      {   // 490 & 830 with same information
+        "title": "Perspectives in Continental philosophy series; no. 39",
+        "partOf": [
+          {
+            "id": "abcdefg",
+            "identifiers": [
+              {
+                "identifierType": {
+                  "id": "issn",
+                  "label": "ISSN",
+                  "type": "IdentifierType"
+                  },
+                "value": "1089-3938",
+                "type": "Identifier"
+              }
+            ],
+            "title": "Perspectives in Continental philosophy series",
+            "type": "Series"
+          }
+        ],
+        "type": "Series"
+      },
+      {  // Second 830 with slighly different information
+        "title": "Fordham perspectives in continental philosophy",
+        "type": "Series"
+      },
+     {   // 830 & 773 contain the same information
+        "title": "ACLS Humanities E-Book.",
+        "type": "Work" //or Series?
```

Comment:
This "//or Series?" raises a question that runs through a lot of this document.

If all we have to go on is a title, how do we determine what kind of thing it refers to?

For unidentified targets, should they have a type of "Unknown" (or no type at all?)

Reply:
4xx and 83x properties are about Series, so they can be identified as such even in the absence of an ISSN in the x field.

773 is not necessarily a series, but may refer to things "serial in nature", (a rose by any other name). So I would guess that if we get a matching 830 and 773, then the 830 "wins", and it's a series.

That leaves the scenario where we have 773 on its own, and no id.  I think in that situation, we can't call it a Work, because we can't find the work.  We shouldn't call it a series, because it may not be, it might just be a single book published in three volumes (or even just a single volume, 773 isn't picky).